### PR TITLE
Update german translation for V10.1.2

### PR DESCRIPTION
**Pull request by Henry2o1o**

### CODE REVIEW

File: src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml [hunk 1]
```diff
@@ -281,7 +281,7 @@
     <sys:String x:Key="Activate_Product_Key">Produktschlüssel</sys:String>
     <sys:String x:Key="Activate_Product_Key.Description">Office-Produktschlüssel installieren, Produktschlüsselformat: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
     <sys:String x:Key="Activate_Install_Product_Key">Schlüssel _installieren</sys:String>
-    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Uninstall all unactivated keys</sys:String>
+    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Alle nicht aktivierten Schlüssel deinstallieren</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml:284]: The access key `A` is same as the next string, please change to other one if possible.

File: src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml [hunk 2]
```diff
@@ -298,41 +298,41 @@
...
-    <sys:String x:Key="Activate_License_Lic.Period">Remaining period:</sys:String>
-    <sys:String x:Key="Activate_License_Lic.Error_Code">Error code:</sys:String>
-    <sys:String x:Key="Activate_License_Lic.Error_Description">Error description:</sys:String>
-    <sys:String x:Key="Activate_License_VL.Type">Activation type:</sys:String>
+    <sys:String x:Key="Activate_Licenses">Office-Lizenzen</sys:String>
+    <sys:String x:Key="Activate_License">Lizenzen Liste</sys:String>
+    <sys:String x:Key="Activate_License.Description">Installierte Lizenzen verwalten.</sys:String>
+    <sys:String x:Key="Activate_License_Refresh">_Aktualisieren</sys:String>
+    <sys:String x:Key="Activate_License_Expand">_Alle erweitern</sys:String>
+    <sys:String x:Key="Activate_License_Collapse">_Alle einklappen</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml:306]: The access keys should be same as original. Such as `_Refresh` -> `Aktualisie_ren` and `_Expand all` -> `Alle _erweitern`. If not, you should set a new one according the rules: https://wpf-tutorial.com/control-concepts/access-keys/

**Review by YerongAI — CHANGES_REQUESTED** [review]
Please check all access keys and change them if needed.

**Comment by Berny23** [comment]
This translation attempt is terrible and has clearly been made via copy & paste from an online translator tool. 
Also, strings like "Lizenzen Liste", "Intervall erneuern:" or "Aktivierung Objektname:" are all literal translations which are grammatically incorrect and don't make any sense in context.

**Comment by iDolmatov** [comment]
https://github.com/YerongAI/Office-Tool/commits/main/src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml
The main translator has more priority and credibility. It makes sense to reject the pull request.

**Comment by YerongAI** [comment]
As @Berny23 pointed out. The translation is not perfect yet.

Our projects are intended to provide quality translations to our users. If not, or if only machine translation is used, then our project loses its meaning.

We hope you can learn more from it and providing better quality translations in the future. Thanks.

**Pull request closed without merging**